themes/default/content/community/puluminaries/_index.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Puluminaries
+meta_desc: The Developer Advocate Team at Pulumi is here to help you understand cloud engineering to build, deploy, and manage your whole stack. Use this page to reach us.
```

Comment:
meta seems wrong

Reply:
Oh whoops. You're correct. I will fix.

Reply:
Fixed in 29eb231810ac9059c36a337d3464470a2f3f1785